moveit_kinematics/kdl_kinematics_plugin/src/chainiksolver_pos_nr_jl_mimic.cpp

```diff
@@ -60,6 +60,12 @@ ChainIkSolverPos_NR_JL_Mimic::ChainIkSolverPos_NR_JL_Mimic(const Chain& _chain,
   ROS_DEBUG_NAMED("kdl", " ");
 }
 
+void ChainIkSolverPos_NR_JL_Mimic::updateInternalDataStructures()
+{
+  // TODO: move (re)allocation of any internal data structures here
+  // to react to changes in chain
```

Comment:
is this really required for the melodic build?

Reply:
Yes, it is a new abstract function in KDL 1.4, which makes the compiler complain about instantiating abstract classes...

Reply:
great, got it